src/pages/index.tsx

```
@@ -166,11 +166,11 @@ const TabBox = ({icon, label, description, list, link}: any) => {
     <div>
       <div className={styles.tabBox}>
-        <div className={styles.heading}>
-          {icon && <Icon className={styles.leftIcon} btn="left" icon={icon} size='inherit' color='var(--teal-600)' /> }
-          <span>{label}</span>
-        </div>
         <div className={styles.content}>
+          <div className={styles.heading}>
+            {icon && <Icon className={styles.leftIcon} btn="left" icon={icon} size='inherit' color='var(--teal-600)' /> }
+            <span>{label}</span>
+          </div>
           <p dangerouslySetInnerHTML={{__html: description}}></p>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)